index.mjs

```diff
@@ -7,6 +7,15 @@ if (typeof process !== 'undefined') {
   global.Worker = function() { return { postMessage: function() { } } }
 }
 
+function kebabToCamel(attribute) {
```

Comment:
Do we need to account for `snake_case` or `PascalCase` here as well? Or will those work as expected without us intervening 🤔 

For example, would `my_attributeChanged` or `MyAttributeChanged` work currently?

Reply:
That's a good question. `snake_case` will work as written but `PascalCase` is completely broken but it is broken at the platform level not our code. Pascal/Camel case attributes never fire the `attributeChangedListener` see: https://enhance.dev/docs/elements/state/attributes#attribute-names

If and when this gets merged, I'll update our docs at https://enhance.dev/docs/conventions/components

Reply:
Oh right, duh. I even edited that very doc myself recently. Tea has not kicked in yet. Whee! 🤦🏻 

Sounds good!